backend/src/nodes/properties/outputs/numpy_outputs.py

```diff
@@ -96,10 +96,18 @@ def get_broadcast_data(self, value: np.ndarray):
         img = value
         h, w, c = get_h_w_c(img)
 
-        base64_img = preview_encode(img, 512)
+        # Encode for multiple scales. Use the preceding scale to save time encoding the smaller sizes.
+        base64_2048, np_2048 = preview_encode(img, 2048)
+        base64_1024, np_1024 = preview_encode(np_2048, 1024)
+        base64_512, np_512 = preview_encode(np_1024, 512)
+        base64_256, _ = preview_encode(np_512, 256)
+        del np_2048, np_1024, np_512
```

Comment:
Why do you do this?

Reply:
Just in case... I don't really know how pythons garbage collection works

Reply:
Unless you [`gc.collect` afterwards](https://stackoverflow.com/questions/1316767/how-can-i-explicitly-free-memory-in-python), it won't help much. It's like we need that those relatively small images anyway. GC should be smart enough to clean those up in a timely manner.